app/views/idv/shared/_doc_capture_interstitial.html.erb

```diff
@@ -8,18 +8,30 @@
       ) %>
 <% end %>
 
+<div class="text-center margin-bottom-4">
+  <%= image_tag asset_url('idv/interstitial_icons.svg'), width: 207, height: 88, alt: '', class: 'display-block margin-x-auto' %>
+</div>
+
 <%= render PageHeadingComponent.new do %>
-  <%= t('doc_auth.headings.verify_with_phone') %>
+  <%= t('doc_auth.headings.document_capture') %>
 <% end %>
 
 <% if decorated_sp_session.sp_name.present? %>
   <p>
-    <%= t('doc_auth.info.login_access_sp', app_name: APP_NAME, sp_name: decorated_sp_session.sp_name) %>
+    <%= t('doc_auth.info.socure_hybrid_flow', app_name: APP_NAME, sp_name: decorated_sp_session.sp_name) %>
```

Comment:
Does the doc_auth.info.socure_hybrid_flow text string only appear if the user is coming from a computer (hybrid flow)? If a user starts the process on their phone and sees this interstitial, then that part of the text should not appear. Screenshot of mobile flow or link to test would be great, thanks!